citizen/run_citizen_mainnet.sh

```diff
@@ -6,7 +6,7 @@ echo "Run citizen with port 7100..."
 loopchain -o ${CONF_DIR}/citizen_mainnet.json &
 
 echo "Run iconservice for citizen 7100 start!"
-iconservice start -c ${CONF_DIR}/iconservice_mainnet.json &
+iconservice start -c ${CONF_DIR}/iconservice_mainnet.json
```

Comment:
Any reason to stop running the process as a daemon?

Reply:
`iconservice` runs in the background by default. `iconrpcserver` also.